projects/kit/components/input-range/input-range.style.less

```diff
@@ -1,143 +1,52 @@
-@import 'taiga-ui-local';
-
-@padding: 1rem;
-@padding2: @padding * 2;
-
-:host {
-    .createStackingContext();
-    display: block;
-    font: var(--tui-font-text-m);
-    border-radius: var(--tui-radius-m);
-    color: var(--tui-text-01);
-
-    &._segmented {
-        // TODO: research if it can be used in rem units
-        /* stylelint-disable-next-line */
-        border-bottom: 26px solid transparent;
-    }
-
-    &[data-mode='onDark'] {
-        color: var(--tui-base-01);
+tui-input-number ::ng-deep tui-wrapper {
+    &:after {
+        border: none;
     }
 }
 
-.t-native-wrapper {
-    .fullsize();
+.t-wrapper {
     display: flex;
 }
 
-.t-pluralize {
-    display: flex;
-    align-items: center;
-    padding-right: 0.75rem;
-    margin-left: -0.5rem;
-
-    &_right {
-        margin-left: -0.75rem;
-
-        :host[data-size='l'] & {
-            padding-top: 1.25rem;
-        }
-
-        :host._disabled & {
-            color: var(--tui-text-03);
-        }
-
-        :host[data-mode='onDark']._disabled & {
-            color: var(--tui-text-03-night);
-        }
-    }
-}
-
-.t-native {
-    .clearinput();
+.t-text-input {
     flex: 1;
```

Comment:
Left input moves the right one away with a long placeholder. I wonder if it makes sense to absolutely position inputs instead and make first one have 100% of width and the second to just overlay on top with 50% right: 0
![image](https://user-images.githubusercontent.com/11832552/162413774-32ee7e76-5fdd-4b8d-ad79-824eb6f32f5a.png)

Reply:
good point, thanks!

But if make them absolutely positioned, we can face with problem of overlapping long values.
May be we should make `max-width: 80%` for left input ?

Reply:
That's up to you to test extreme values/placeholders and pick best solution. In my head absolute positioning causes less troubles :)

Reply:
fixed ✅

Also, I added screenshot tests for this case
![01-long-placeholder_basic snap](https://user-images.githubusercontent.com/35179038/162719785-f9f77220-2163-4e52-b187-a67117608d81.png)

___
![02-long-placeholder_value-content snap](https://user-images.githubusercontent.com/35179038/162719829-67bed378-0e1f-4cf4-8bb6-c5001d8d762f.png)

___
![03-long-placeholder_pluralize snap](https://user-images.githubusercontent.com/35179038/162719847-a0ab0d5f-adee-47c2-8d08-ec51dc363a2e.png)